rpc_jobs/ops_fabric_chatbot.yml

```diff
@@ -0,0 +1,51 @@
+- job:
+    name: ops-fabric-chatbot-gate
+    description: "lint and unit tests for ops-fabric-chatbot"
+    disabled: false
+    display-name: "ops-fabric-chatbot lint and unit tests"
+    project-type: workflow
+    concurrent: true
+    properties:
+      - github:
+          url: https://github.com/rcbops/ops-fabric-chatbot/
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+    triggers:
+        - github-pull-request:
```

Comment:
The spacing for the github-pull-request section looks like it's over two spaces more than it needs to be.